src/functions.cpp

```diff
@@ -141,10 +141,28 @@ void removeFile(std::string_view filename)
     }
 }
 
+/**
+ * \brief Get the current timestamp as a string.
+ *
+ * This function generates the current timestamp in the format
+ * "YYYYMMDD_HHMMSS".
+ *
+ * \return A string with the current timestamp.
+ */
+std::string getCurrentTimestamp()
+{
+    auto now = std::chrono::system_clock::now();
+    auto time_t_now = std::chrono::system_clock::to_time_t(now);
+    std::stringstream ss;
+    ss << std::put_time(std::localtime(&time_t_now), "%Y%m%d_%H%M%S");
```

Comment:
Instead of manually creating a model what about following the ISO 8601? I think the language already supports it with `"%FT%TZ"`.

There's some references here:
https://stackoverflow.com/questions/9527960/how-do-i-construct-an-iso-8601-datetime-in-c
https://en.wikipedia.org/wiki/ISO_8601
https://en.cppreference.com/w/c/chrono/localtime